pylib/anki/media.py

```diff
@@ -159,7 +163,11 @@ def strip(self, txt: str) -> str:
         return txt
 
     def escapeImages(self, string: str, unescape: bool = False) -> str:
-        "Apply or remove percent encoding to image filenames."
+        "escape_media_filenames alias for compatibility with add-ons."
+        return self.escape_media_filenames(string, unescape)
```

Comment:
Thanks for making those changes. For future reference, you can do this more succinctly with a variable:

https://github.com/ankitects/anki/blob/e3eca1c5637b6b5e285e11c69d50623d4bb85d27/pylib/anki/decks.py#L368